### ISSUE TITLE
K2A Offline crashes when trying to create new entry

### ISSUE THREAD

**Issue by bege10** [PhilippC/keepass2android#2292]
K2A Offline crashes the very moment I tap in the name field of the new entry.

Scoop displays these crash logs
- Create manually
```
FATAL EXCEPTION: main
Process: keepass2android.keepass2android_nonet, PID: 8004
android.runtime.JavaProxyThrowable: System.ArgumentException: An item with the same key has already been added. Key: keepass2android.services.AutofillBase.ViewNodeInputField
  at System.Collections.Generic.Dictionary`2[TKey,TValue].TryInsert (TKey key, TValue value, System.Collections.Generic.InsertionBehavior behavior) [0x0015a] in <8c1f40060880450f83bb90f0629b1667>:0 
  at System.Collections.Generic.Dictionary`2[TKey,TValue].Add (TKey key, TValue value) [0x00000] in <8c1f40060880450f83bb90f0629b1667>:0 
  at Kp2aAutofillParser.StructureParserBase`1[FieldT].Parse (System.Boolean forFill, System.Boolean isManualRequest, Kp2aAutofillParser.AutofillView`1[TField] autofillView) [0x002d2] in <9d921e539c20416fbc81d0c3c5afc0c5>:0 
  at keepass2android.services.AutofillBase.StructureParser.Parse (System.Boolean forFill, System.Boolean isManualRequest, Kp2aAutofillParser.AutofillView`1[TField] autofillView) [0x00000] in <2c79b7425a854bebbd39062e5dbd2d83>:0 
  at keepass2android.services.AutofillBase.StructureParser.ParseForFill (System.Boolean isManual) [0x00018] in <2c79b7425a854bebbd39062e5dbd2d83>:0 
  at keepass2android.services.AutofillBase.AutofillServiceBase.OnFillRequest (Android.Service.Autofill.FillRequest request, Android.OS.CancellationSignal cancellationSignal, Android.Service.Autofill.FillCallback callback) [0x0009f] in <2c79b7425a854bebbd39062e5dbd2d83>:0 
  at Android.Service.Autofill.AutofillService.n_OnFillRequest_Landroid_service_autofill_FillRequest_Landroid_os_CancellationSignal_Landroid_service_autofill_FillCallback_ (System.IntPtr jnienv, System.IntPtr native__this, System.IntPtr native_request, System.IntPtr native_cancellationSignal, System.IntPtr native__callback) [0x00020] in <e74dcf35373743e2bbf871ba34f3d1e2>:0 
  at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PPLLL_V (_JniMarshal_PPLLL_V callback, System.IntPtr jnienv, System.IntPtr klazz, System.IntPtr p0, System.IntPtr p1, System.IntPtr p2) [0x00005] in <e74dcf35373743e2bbf871ba34f3d1e2>:0 
	at crc6402e1bf64a3331322.AutofillServiceBase.n_onFillRequest(Native Method)
	at crc6402e1bf64a3331322.AutofillServiceBase.onFillRequest(AutofillServiceBase.java:42)
	at android.service.autofill.-$$Lambda$I0gCKFrBTO70VZfSZTq2fj-wyG8.accept(Unknown Source:8)
	at com.android.internal.util.function.pooled.PooledLambdaImpl.doInvoke(PooledLambdaImpl.java:307)
	at com.android.internal.util.function.pooled.PooledLambdaImpl.invoke(PooledLambdaImpl.java:201)
	at com.android.internal.util.function.pooled.OmniFunction.run(OmniFunction.java:97)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7664)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

- Create via K2A request after filling credentials in Firefox for Android

```
FATAL EXCEPTION: main
Process: keepass2android.keepass2android_nonet, PID: 25088
android.runtime.JavaProxyThrowable: System.NullReferenceException: Object reference not set to an instance of an object
  at keepass2android.services.AutofillBase.StructureParser.Parse (System.Boolean forFill, System.Boolean isManualRequest, Kp2aAutofillParser.AutofillView`1[TField] autofillView) [0x0008e] in <2c79b7425a854bebbd39062e5dbd2d83>:0 
  at keepass2android.services.AutofillBase.StructureParser.ParseForSave () [0x00018] in <2c79b7425a854bebbd39062e5dbd2d83>:0 
  at keepass2android.services.AutofillBase.AutofillServiceBase.OnSaveRequest (Android.Service.Autofill.SaveRequest request, Android.Service.Autofill.SaveCallback callback) [0x0002b] in <2c79b7425a854bebbd39062e5dbd2d83>:0 
  at Android.Service.Autofill.AutofillService.n_OnSaveRequest_Landroid_service_autofill_SaveRequest_Landroid_service_autofill_SaveCallback_ (System.IntPtr jnienv, System.IntPtr native__this, System.IntPtr native_request, System.IntPtr native__callback) [0x00017] in <e74dcf35373743e2bbf871ba34f3d1e2>:0 
  at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PPLL_V (_JniMarshal_PPLL_V callback, System.IntPtr jnienv, System.IntPtr klazz, System.IntPtr p0, System.IntPtr p1) [0x00005] in <e74dcf35373743e2bbf871ba34f3d1e2>:0 
	at crc6402e1bf64a3331322.AutofillServiceBase.n_onSaveRequest(Native Method)
	at crc6402e1bf64a3331322.AutofillServiceBase.onSaveRequest(AutofillServiceBase.java:50)
	at android.service.autofill.-$$Lambda$KrOZIsyY-3lh3prHWFldsWopHBw.accept(Unknown Source:6)
	at com.android.internal.util.function.pooled.PooledLambdaImpl.doInvoke(PooledLambdaImpl.java:292)
	at com.android.internal.util.function.pooled.PooledLambdaImpl.invoke(PooledLambdaImpl.java:201)
	at com.android.internal.util.function.pooled.OmniFunction.run(OmniFunction.java:97)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7664)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

**Comment by PhilippC** [comment]
which app version is this referring to? Can you please make sure you're running https://github.com/PhilippC/keepass2android/releases/tag/v1.09e-r5?

**Comment by bege10** [comment]
I am on r3. I update from IzzyOnDroid. The mentioned version is not available there.

**Comment by PhilippC** [comment]
I'm closing this as this bug is fixed in 1.09e-r5
